en/docs/release-notes/version-3.3.0.md

```diff
@@ -37,6 +37,4 @@ If you are new to Ballerina, you can download the [installers](/downloads/#swanl
 
 - This release includes an experimental feature that allows you to visualize service interactions in your project.
 - It also comes with a view to see record-type compositions and relationship.
-- Use the "Ballerina: Project Design" command to open design view.
-
-Apart from the above, there are lots of other bug fixes and improvements. For more details, see the [issue list](https://github.com/wso2/ballerina-plugin-vscode/issues?q=is%3Aissue+is%3Aclosed). 
```

Comment:
Why did we remove the bug fixes list?

Reply:
Removed based on https://github.com/wso2/docs-ballerina-visual-studio-code-plugin/pull/19#discussion_r1065516346.

Reply:
ack.